plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeSystemTables.java

```diff
@@ -90,4 +90,16 @@ public void testHistoryTable()
             assertUpdate("DROP TABLE IF EXISTS test_checkpoint_table");
         }
     }
+
+    @Test
+    public void testPropertiesTable()
+    {
+        try {
+            assertUpdate("CREATE TABLE test_simple_table (_bigint BIGINT) with (change_data_feed_enabled = true, checkpoint_interval = 5)");
```

Comment:
Please don't reuse the same table name. This name is used in `testHistoryTable` method.